src/components/ClaimCard/ClaimCard.tsx

```diff
@@ -0,0 +1,131 @@
+import { CSS } from '../../stitches.config';
+import { Box, Text, ArrowDiagonalIcon, Button } from '../../ui';
+
+//#region Interfaces & Types
+
+type BaseStyledComponents = {
+  css?: CSS;
+};
+
+export interface ClaimCardProps extends BaseStyledComponents {
+  epochInfo: EpochInfoSectionProps[];
+  actionSection: ActionSectionProps;
+  children?: React.ReactNode;
+}
+
+export interface EpochInfoSectionProps extends BaseStyledComponents {
+  title: string;
+  subTitle: string;
+  giveInfo: string;
+}
+
+export interface ActionSectionProps extends BaseStyledComponents {
+  actionText1: string;
+  actionText2: string;
+  onClaim?(): void;
+  onViewHistory?(): void;
+}
+
+//#endregion Interfaces
+
+//#region Molecules
+export const EpochInfoSection: React.FC<EpochInfoSectionProps> = (
+  props
+): JSX.Element => {
+  return (
+    <Box
+      css={{
+        display: 'flex',
+        flexDirection: 'column',
+        gap: '$xs',
+        ...props.css,
+      }}
+    >
+      <Text
+        css={{
+          fontSize: '$5',
+          color: '$neutralGrayDark',
+          fontWeight: '$bold',
+        }}
+      >
+        {props.title}
+      </Text>
+      <Text css={{ fontSize: '$3', color: '$gray400' }}>{props.subTitle}</Text>
+      <Box
+        css={{
```

Comment:
can this fit on one line? if so, please make it a one-liner

Reply:
I'm using prettier with 80 line limit, do you want me to edit that rule in my actual config to do this one-liner?